File: src/question/month08/day0821/gebal_0821.c
#include<stdio.h>

int main() {
	int first[2][4] = {
		{1, 2, 3, 4},
		{5, 6, 7, 8}
	};
	int second[2][4] = {
		{1, 4, 7, 8},
		{3, 6, 9, 8}
	};
	for (int i = 0; i < 2; i++) {
		for (int j = 0; j < 4; j++) {
			printf("%d ", first[i][j] * second[i][j]);
		}
		printf("\n");
	}
}

File: src/question/month08/day0821/gebal_0821.py
import numpy as np
first = np.array([[1, 2, 3, 4], [5, 6, 7, 8]])
second = np.array([[1, 4, 7, 8], [3, 6, 9, 8]])
[[print(v, end=' ') if i != 3 else print(v) for i, v in enumerate(x)] for x in first * second]
